cmd/operator/README.md

```diff
@@ -134,8 +105,23 @@ NAME                     DESIRED   READY   NODE SELECTOR   AGE
 gpudeviceplugin-sample   1         1                       5s
 ```
 
+## Upgrade
+
+The upgrade of the deployed plugins can be done by simply installing a new release of the operator.
+
+The operator auto-upgrades operator-managed plugins (CR images and thus corresponding deployed daemonsets) to the current release of the operator.
+
+During upgrade the tag in the image path is updated (e.g. docker.io/intel/intel-sgx-plugin:tag), but the rest of the path is left intact.
+
+No upgrade is done for:
+
+- Non-operator managed deployments
+- Operator deployments without numeric tags
```

Comment:
should not be possible because the validating webhook checks the tags *are* semver tags

Reply:
Remove this completely? It was there originally, so I'm not sure.

Reply:
let's keep it then

Reply:
Restored the snippet.